src/compiler/types.ts

```diff
@@ -6718,6 +6718,7 @@ namespace ts {
         readonly importModuleSpecifierEnding?: "auto" | "minimal" | "index" | "js";
         readonly allowTextChangesInNewFiles?: boolean;
         readonly providePrefixAndSuffixTextForRename?: boolean;
+        readonly stringLiteralReplacementMode?: boolean;

```

Comment:
Also if this is really needed it shouldn't say node.. instead may be `provideStringLiteralReplacementSpan`